testsuite/dynaconf_loader.py

```diff
@@ -61,6 +61,19 @@ def _guess_version(ocp):
     return str(version)
 
 
+def _guess_apicast_version(ocp):
+    """Attempt to determine version of apicast operator from subscription"""
+
+    version = None
+    try:
+        version = ocp.get_apicast_operator().object().status.installedCSV.split(".v")[1]
```

Comment:
Will this return version 2.12 or 0.5.3?

Reply:
version of the operator not 3scale: `installedCSV: apicast-operator.v0.5.2`

Reply:
In that case why do you fallback to testsujte version which has different versioning?